__tests__/api.test.js

```diff
@@ -31,3 +32,60 @@ describe('GET/api/topics', () => {
         });
     });
 });
+
+
+describe('GET/api', () => {
+    it('200: responds with all endpoints available', () => {
+        return request(app).get('/api').expect(200);
+    })   
+
+    it('200: responds with the expected JSON describing all available endpoints', () => {
+        return request(app).get('/api')
+        .then((response) => {
+        expect(response.body['GET /api'].description).toBe('serves up a json representation of all the available endpoints of the api');
+        expect(response.body['GET /api/topics'].description).toBe('serves an array of all topics');
+        expect(response.body['GET /api/articles'].description).toBe('serves an array of all articles');
+        })   
+    });
+});
+
+describe('GET/api/articles/:article_id', () => {
+    it('200: responds with an article object', () => {
+        return request(app).get('/api/articles/1').expect(200);
+    });   
+
+    it('200: responds with an article object with id of 1', () => {
+        return request(app).get('/api/articles/1')
+        .then((response) => {
+            const article = response.body.article
+            expect(article).toHaveProperty('author', expect.any(String));
+            expect(article).toHaveProperty('title', expect.any(String));
+            expect(article).toHaveProperty('article_id', expect.any(Number));
+            expect(article).toHaveProperty('body', expect.any(String));
+            expect(article).toHaveProperty('topic', expect.any(String));
+            expect(article).toHaveProperty('created_at', expect.any(String));
+            expect(article).toHaveProperty('votes', expect.any(Number));
+            expect(article).toHaveProperty('article_img_url', expect.any(String));
+        })
+    });
+
+    it('400: responds with an error message when passed an article_id that is not accepted', () => {
+        return request(app).get('/api/articles/hello')
+        .then((result) => {
```

Comment:
to make sure `express` is giving a `400` status here, you can write `...get('/api/articles/hello').expect(400).then(...`